CONTRIBUTE

```diff
@@ -0,0 +1,69 @@
+CONTRIBUTING TO TQDM
```

Comment:
What do you mean? There are two sides to make a release: update the version using semantic versioning, and uploading to pypi when we want to make a new release. Have you got something else in mind?